src/components/Offers/Offers.tsx

```diff
@@ -49,32 +72,52 @@ const Offers = () => {
    }
 
    return (
-      <section className="flex flex-col w-[877px] h-screen p-10 bg-gray-lightest gap-[56px]">
-         <div className="flex gap-3 " ref={ref}>
+      <section className=" w-[877px] h-screen p-10 bg-gray-lightest gap-[56px]">
+         <div className="flex gap-3 mb-6" ref={ref}>
             <SearchWrapper>
                <SearchInput
                   variant={SearchVariant.TITLE}
                   placeholder="Search for"
                   value={searchTerm}
-                  onChange={handleTitleChange}
-                  onFocus={() => setShowTitleSuggestion(true)}
+                  onChange={handleInputChange}
+                  onFocus={handleInputFocus}
                />
-               {showTitleSuggestion && titleSuggestions.length > 0 && (
-                  <ul className="absolute top-full translate-y-[2px] left-0 w-full max-h-[188px] flex flex-col shadow-checkboxShadow overflow-y-auto hiddenScrollbar">
-                     {titleSuggestions.map(offer => (
-                        <li
-                           className="flex justify-between items-center px-4 py-3 border border-gray-light bg-white text-gray-dark cursor-pointer"
-                           key={offer._id}
-                           onClick={() => suggestionClick(offer.title)}>
-                           {highlightText(offer.title, searchTerm.title)}
-                           <p className="text-reg-12 ">{offer.companyName}</p>
-                        </li>
-                     ))}
-                  </ul>
+               {showSuggestions[SearchVariant.TITLE] && suggestions.length > 0 && (
+                  <SuggestionsList
+                     suggestions={suggestions}
+                     variant={SearchVariant.TITLE}
+                     searchTerm={searchTerm[SearchVariant.TITLE]}
+                     handleSuggestionClick={handleSuggestionClick}
+                  />
+               )}
+            </SearchWrapper>
+            <SearchWrapper>
+               <SearchInput
+                  variant={SearchVariant.CITY}
+                  placeholder="Search location"
+                  value={searchTerm}
+                  onChange={handleInputChange}
+                  onFocus={handleInputFocus}
+               />
+               {showSuggestions[SearchVariant.CITY] && suggestions.length > 0 && (
```

Comment:
**nitpick:** Make const to avoid duplication `const isAnySuggestion = suggestions.length > 0`